test/cri-containerd/main.go

```diff
@@ -22,8 +22,8 @@ const (
 	lcowRuntimeHandler           = "lcow-debug"
 	imageWindowsRS5Nanoserver    = "mcr.microsoft.com/windows/nanoserver:1809"
 	imageWindowsRS5Servercore    = "mcr.microsoft.com/windows/servercore:1809"
-	imageLcowK8sPause            = "k8s.gcr.io/pause"
-	imageLcowAlpine              = "docker.io/library/alpine"
+	imageLcowK8sPause            = "k8s.gcr.io/pause:3.1"
```

Comment:
Is this just hygiene? Or was it breaking something to not specify the tag?

Reply:
Turns out `:latest` was not `:3.1` but in fact a much older image. So the test would pull an image that is not the one that the sandbox needs. So the first sandbox activation will always fail on a clean install.